tsconfig.build.json

```diff
@@ -5,12 +5,7 @@
   "files": [],
   "include": [],
   "references": [
-    { "path": "./packages/integration-testsuite/" },
-    { "path": "./packages/plugin-landing-page-graphql-playground" },
-    { "path": "./packages/plugin-landing-page-graphql-playground/tsconfig.cjs.json" },
-    { "path": "./packages/plugin-response-cache" },
-    { "path": "./packages/plugin-response-cache/tsconfig.cjs.json" },
-    { "path": "./packages/server" },
-    { "path": "./packages/server/tsconfig.cjs.json" },
+    { "path": "./tsconfig.build.esm.json" },
```

Comment:
Weird fact: reversing the order of these two lines gives a situation where in a clean checkout, `npm run compile` gives an error the first time you run it and succeeds the second time. 🤷